autoscale/src/main/java/com/sequenceiq/periscope/monitor/event/ScalingEvent.java

```diff
@@ -78,4 +80,12 @@ public ScalingAdjustmentType getScalingAdjustmentType() {
     public void setScalingAdjustmentType(ScalingAdjustmentType scalingAdjustmentType) {
         this.scalingAdjustmentType = scalingAdjustmentType;
     }
+
+    public Long getActivityId() {
+        return activityId;
+    }
+
+    public void setActivityId(Long activityId) {
```

Comment:
Are we making sure this is always set ?
CronTimeEvaluator does not seem to be setting this.

Reply:
Updated CronTimeEvaluator to set this value